code/qcommon/ojk_saved_game_class_archivers.h

```diff
@@ -19,10 +19,7 @@ template<>
 class SavedGameClassArchiver<cplane_t>
 {
 public:
-	static constexpr bool is_implemented()
-	{
-		return true;
-	}
+	enum { is_implemented = true };
```

Comment:
The main reason for doing this is described here:

http://stackoverflow.com/questions/2172647/template-metaprogramming-difference-between-using-enum-hack-and-static-const